### PR TITLE
Closes #609 adds text DAU or Total next to event

### LINKED ISSUE #609: Add (DAU) to hover if graph is filtered on DAU
**Is your feature request related to a problem? Please describe.**
When hovering over the graph it's not clear whether the event shows total volume of DAU

image

**Describe the solution you'd like**

Show `(DAU)` next to event

### PR DESCRIPTION
**Pull request by abhijitghate**
## Changes

- added a few lines to add text next to event in DAU

## Checklist

- [ ] All querysets/queries filter by Team (if applicable)
- [ ] Backend tests (if applicable)
- [ ] Cypress E2E tests (if applicable)

### CODE REVIEW

File: frontend/src/scenes/trends/LineGraph.js [hunk 1]
```diff
@@ -188,6 +188,12 @@ export function LineGraph({
                                       if (entityData.dotted && !(tooltipItem.index === entityData.data.length - 1))
                                           return null
                                       var label = entityData.chartLabel || entityData.label || ''
+                                      if (entityData.action) {
+                                          let math = entityData.action.math
```
Comment (EDsCODE) [inline frontend/src/scenes/trends/LineGraph.js:192]: Thanks for this! One more thing to resolve is that we default the math to total so there are instances where `entityData.action.math` is null and not necessarily 'total'.

Comment (abhijitghate) [reply frontend/src/scenes/trends/LineGraph.js:192]: Oh okay! I will update it.

**Comment by abhijitghate** [comment]
Hey @EDsCODE ! Seems like I messed somthing up. Could you please guide me here? What steps should I take to fix these failures?

**Comment by EDsCODE** [comment]
We have a few spotty Cypress tests! I reran the tests and it looks ok

**Comment by abhijitghate** [comment]
Oh okay! Thanks!:)

**Review by EDsCODE — APPROVED** [review]
lgtm

**Pull request merged**